code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -505,6 +505,11 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks)
 				. = ("You deftly [pick("spin", "twirl")] [src] managing to keep all the contents inside.")
 				if(!ON_COOLDOWN(user, "bartender spinning xp", 180 SECONDS)) //only for real cups
 					JOB_XP(user, "Bartender", 1)
+			if (istype(src, /obj/item/reagent_containers/food/drinks/cola))
+				var/obj/item/reagent_containers/food/drinks/cola/soda_can = src
+				if (soda_can.is_sealed && soda_can.reagents.has_reagent("cola", 5))
```

Comment:
Consider the following additional reagents: "sodawater", "tonic"

Reply:
Sounds good! I'll add those.